authors/apps/authentication/serializers.py

```diff
@@ -214,55 +215,129 @@ class SocialAuthSerializer(serializers.Serializer):
         allow_blank=True,
         default=""
     )
+
+
 class PasswordResetSerializer(serializers.Serializer):
     """
     Serializer for password reset
     """
-    email = serializers.EmailField(required=True)
+    email = serializers.EmailField(
+        required=False,
+        allow_blank=True,
+        allow_null=True,
+        write_only=True
+    )
 
-    def validate_email(self, value):
+    def validate(self, data):
+        email = data.get("email")
+        if not email:
+            raise serializers.ValidationError({
+                "email": "Please provide your email address"
+            })
         try:
-            EmailValidator(value)
+            EmailValidator(email)
         except ValidationError as e:
```

Comment:
This edge case here is not tested in your code...Try testing for an invalid email to improve the tests.

Reply:
Hello @IssaIan ,
It is already tested. It is the first test in test_password_reset.py file